int8 flash_count
---
string response
